app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -373,3 +374,75 @@ def test_internal_only_does_not_add_comment(self):
         self.mock_bugzilla_requests_post.assert_not_called()
 
         self.assertEqual(Notification.objects.count(), 0)
+
+
+class TestUpdateExperimentStatus(
+    MockRequestMixin, MockNormandyMixin, TestCase
+):
+
+    def setUp(self):
+        super().setUp()
+
+        self.experiment = ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_ACCEPTED, normandy_id=1234
+        )
+
+    def test_successful_experiment_status_update(self):
```

Comment:
I think you need a couple extra test cases here, something like

    test_accepted_experiment_becomes_live_if_normandy_enabled
    test_accepted_experiment_stays_accepted_if_normandy_disabled
    test_live_experiment_statys_live_if_normandy_enabled
    test_live_experiment_becomes_complete_if_normandy_disabled

that sets up a mock experiment in each state and the mock normandy response and checks the status at the end.  

Also probably check that the right changelog is created?